### ISSUE TITLE
Content processor issue

### ISSUE THREAD

**Issue by enriko-riba** [MonoGame/MonoGame#2715]
I wanted to test the dev build (from today 773). Seems that the content processor has a issue with non power of two images. My terrain heightmaps are 129x129 bitmaps. After rebuilding the test project with the monogame dev build, the heightmap seems to be corrupted. Note the diagonal pattern on the terrain:
![untitled](https://cloud.githubusercontent.com/assets/1684961/3278255/23c88b7a-f3b2-11e3-8503-6185b1d90c3e.jpg)
Such patterns are usually the result of one ore more pixels shifted.
Long story short, tried enabling the "resize to power of two" option and the terrain looked normal again. 
Also there might be a problem with the premultiplied alpha or color key. The reddish light billboard in the top mid is supposed to be fully transparent except for central circel - but I need to investigate that further.

**Comment by Jjagg** [comment]
I think this is very likely fixed. There have been multiple PRs related to pixel data alignment. Unless the issue is in the decoder. We should add unit tests for 8bpp BMP.
